### ISSUE TITLE
aiohttp - Unclosed client session (console mess)

### ISSUE THREAD

**Issue by Amelia-exe** [Amelia-exe/RageBot-v3#25]
Unsightly in console, need to fix.

```
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x000001AEFDC108B0>
```

Need to find a way to fix issue / suppress warning to not show.

**Comment by Amelia-exe** [comment]
@Dan6erbond @HamzahSarmad anyone got a fix for this issue?
![image](https://user-images.githubusercontent.com/68903836/92657204-4f9b4180-f2ec-11ea-8e2f-b0ac07e55e8c.png)

**Comment by Amelia-exe** [comment]
#42 some work done on the suppression here.
Need some fresh eyes looking at it to finish it off.

**Comment by Amelia-exe** [comment]
Issue is now resolved ! #43 
@Dan6erbond thanks for the assist 👍🏻
